cf/mixin/fielddomainlist.py

```diff
@@ -0,0 +1,339 @@
+
+                The condition is satisfied if any of its data values
+                equals the value provided.
+
+                If no conditions nor identities (see the *identities*
+                parameter) are provided then all list elements are
+                selected.
+
+                *Parameter example:*
+                  ``longitude=180.0``
+
+                *Parameter example:*
+                  ``time=cf.dt('1959-12-16')``
+
+                *Parameter example:*
+                  ``latitude=cf.ge(0)``
+
+                *Parameter example:*
+                  ``latitude=cf.ge(0), air_pressure=500``
+
+                *Parameter example:*
+                  ``**{'latitude': cf.ge(0), 'long_name=soil_level': 4}``
+
+            OR: `bool`, optional
+                If True then return `True` if at least one metadata
+                construct matches at least one of the criteria given
+                by the *identities* or *conditions* arguments. By
+                default `True` is only returned if the field
+                constructs matches each of the given criteria.
+
+            mode: deprecated at version 3.1.0
+                Use the *OR* parameter instead.
+
+            constructs: deprecated at version 3.1.0
+
+        :Returns:
+
+            `bool`
+                The matching field constructs.
+
+        **Examples:**
+
+            TODO
+
+        """
+        if identities:
+            if identities[0] == "or":
+                _DEPRECATION_ERROR_ARG(
+                    self,
+                    "select_by_construct",
+                    "or",
+                    message="Use 'OR=True' instead.",
+                    version="3.1.0",
+                )  # pragma: no cover
+
+            if identities[0] == "and":
+                _DEPRECATION_ERROR_ARG(
+                    self,
+                    "select_by_construct",
+                    "and",
+                    message="Use 'OR=False' instead.",
+                    version="3.1.0",
+                )  # pragma: no cover
+
+        return type(self)(
+            f
+            for f in self
+            if f.match_by_construct(*identities, OR=OR, **conditions)
+        )
+
+    def select_by_ncvar(self, *ncvars):
+        """Select list elements by netCDF variable name.
+
+        To find the inverse of the selection, use a list comprehension
+        with the `!match_by_ncvar` method of the construct
+        elements. For example, to select all constructs which do *not*
+        have a netCDF name of 'tas':
+
+           >>> gl = cf.{{class}}(
+           ...     f for f in fl if not f.match_by_ncvar('tas')
+           ... )
+
+        .. versionadded:: 3.0.0
+
+        .. seealso:: `select`, `select_by_identity`, `select_by_property`,
+
+        :Parameters:
+
+            ncvars: optional
+                Select constructs from the list. May be one or more:
+
+                  * The netCDF name of a construct.
```

Comment:
A single-item list seems a bit confusing to me, as if something has been cut off. It might be clearer to state it in the sentence, something lke:

```suggestion
                Select constructs from the list. May be one or more netCDF names of constructs.
```

Reply:
There's a similar case of this too in a subsequent method.